inc/geolocation.class.php

```diff
@@ -206,11 +206,13 @@ private function agentGeolocationInputParser(array $input) {
       $dateGeolocation = false;
       $valuesChecked = false;
       if (isset($input['_datetime'])) {
+
+         $dateGeolocation = new DateTime();
+         $dateGeolocation->setTimestamp($input['_datetime']);
```

Comment:
the 2 lines above duplicate the line with createFromFormat()

Reply:
fixed